src/test/java/nl/praegus/fitnesse/responders/testHistory/RecentTestHistoryResponderTest.java

```diff
@@ -39,15 +40,16 @@ public void When_directory_is_not_null_return_historylines_not_sorted() {
     }
 
     @Test
-    public void When_dateTimeFormatter_is_not_working_correctly(){
+    public void checks_if_correct_date_is_shown() {
```

Comment:
2 punten: 
Wanneer is de datum correct?
Er wordt niks getoond. 

Volgens mij is een goede testnaam hier (de volgende keer mogen jullie hem echt zelf verzinnen):
when_the_recent_test_history_is_retreived_the_recent_run_date_is_parsed_correctly

Het gaat erom dat je in je testnaam beschrijft wat je doet (het stukje achter the when) en daarachter wat je dan verwacht dat er gebeurt. En dat is meestal in de vorm van when_i_do_x_then_y_happens.